pkg/orchestrator/provider/docker/container.go

```diff
@@ -103,15 +104,17 @@ func (c *Client) getContainerInfo(ctx context.Context, containerID, imageID stri
 	}
 
 	// Get container image info
-	imageInfo, err := c.getContainerImageInfo(ctx, imageID)
+	imageInfo, err := c.getContainerImageInfo(ctx, info.Image)
 	if err != nil {
 		return models.ContainerInfo{}, err
 	}
 
+	containerName := strings.Trim(info.Name, "/")
```

Comment:
Do you want Trim or TrimSuffix here?

Reply:
It would be `TrimPrefix` in this case, but I think we are fine since docker does not allow container names with slashes